examples/queue/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	marathon "github.com/gambol99/go-marathon"
+)
+
+const marathonURL = "http://127.0.0.1:8080"
```

Comment:
You need to turn this into an non-initialized variable. Should also fix the build error that Travis CI is currently reporting.